kinesisfirehose-logs-extension-demo/README.md

```diff
@@ -0,0 +1,225 @@
+# Centralize log collection with Kinesis Firehose using Lambda Extensions
+
+## Introduction
+
+This pattern walks through an approach to centralize log collection for lambda function with Kinesis firehose using external extensions. The provided code sample shows how to get send logs directly to kinesis firehose without sending them to AWS CloudWatch service.
```

Comment:
Always Lambda....all AWS/Amazon products have capital letters so check these for Lambda/Kinesis/etc.

Reply:
Updated